server/odc-service/src/main/java/com/oceanbase/odc/service/onlineschemachange/model/OnlineSchemaChangeParameters.java

```diff
@@ -56,6 +56,7 @@ public class OnlineSchemaChangeParameters implements Serializable, TaskParameter
 
     private List<String> lockUsers;
     private SwapTableType swapTableType;
+    private Long flowInstanceId;
```

Comment:
Please add `serialVersionUID` manually in this object since you have change the field.

Reply:
Serializable is unnecessary, object be saved as json string , do not save as object byte array

Reply:
add `serialVersionUID` for flow save byte array